### ISSUE TITLE
Some Value fields don't pop up on Firefox Linux

### ISSUE THREAD

**Issue by Sorry-Im-Stupid** [Amorano/Jovimetrix#70]
It's a strange problem and it seems worth it's own issue:
For some reason, the special value fields don't pop up, or have any interaction for me. I don't even know what all to look for, so just tell me any more information you may need and I'll do my best to find it.

![something wrong](https://github.com/user-attachments/assets/355e5600-00ae-4053-9960-dabeb8627dda)

`2024-12-31 14:51:08.718 | ERROR    | Jovimetrix.sup.stream:<module>:36 - NO SPOUT GL SUPPORT

2024-12-31 14:51:08.718 | ERROR    | Jovimetrix.sup.stream:<module>:37 - No module named 'SpoutGL'

2024-12-31 14:51:08.736 | WARNING  | Jovimetrix:__init__:956 - module failed /home/USERNAME/Stable/ComfyUI/ComfyUI/custom_nodes/Jovimetrix/core/device_stream.py

2024-12-31 14:51:08.736 | WARNING  | Jovimetrix:__init__:957 - cannot import name 'window_capture' from 'Jovimetrix.sup.stream' (/home/USERNAME/Stable/ComfyUI/ComfyUI/custom_nodes/Jovimetrix/sup/stream.py)

Failed to extract font properties from /usr/share/fonts/truetype/noto/NotoColorEmoji.ttf: In FT2Font: Can not load face (unknown file format; error code 0x2)

2024-12-31 14:51:09.404 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.create

2024-12-31 14:51:09.405 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.compose

2024-12-31 14:51:09.585 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.calc

2024-12-31 14:51:09.608 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.device_midi

2024-12-31 14:51:09.691 | INFO     | Jovimetrix.sup.shader:<module>:113 -   vertex programs: 0

2024-12-31 14:51:09.691 | INFO     | Jovimetrix.sup.shader:<module>:114 - fragment programs: 13

2024-12-31 14:51:09.694 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.create_glsl

2024-12-31 14:51:09.695 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.utility.info

2024-12-31 14:51:09.695 | WARNING  | Jovimetrix.core.utility.io:<module>:56 - no gifski support

2024-12-31 14:51:09.695 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.utility.io

2024-12-31 14:51:09.696 | INFO     | Jovimetrix:__init__:989 - ✅ Jovimetrix.core.utility.batch

2024-12-31 14:51:09.696 | INFO     | Jovimetrix:__init__:990 - 59 nodes loaded
`

**Comment by Amorano** [comment]
With the WH input, you would need to use a `VALUE` node to input a proper VEC2INT.

Here is a video of what should be happening when you click the vector (or any) numerical field:

https://github.com/user-attachments/assets/77bc278d-8125-4e23-bddf-83fe5c2de756

If that is not happening, that is the first problem, it wont matter that you convert something into a vector.

**Comment by Sorry-Im-Stupid** [comment]
Yeah I'm not getting that, or anything off those ones. The normal fields work like normal, but those split ones do nothing. Same for the color and matte fields. 
And it's the same with your other nodes too, anything with that split layout is non-responsive.

**Comment by Amorano** [comment]
I tried to do some more checking on this, but cant replicate. I guess I need more information about what OS, browser, etc... I will leave this open -- just post more information in the next few days. 

Make sure to use the latest frontend for ComfyUI as well -- cant be sure it isnt a combination of their work + old UI code, etc...

If not, I will just close it as being old UI weirdness.

**Comment by Amorano** [comment]
No new information, and I cant replicate the problem. Just post a new message with a workflow or something and I can re-open.
